ArduCopter/mode_guided.cpp

```diff
@@ -444,6 +451,10 @@ void ModeGuided::pos_control_run()
         // roll, pitch from waypoint controller, yaw heading from GCS or auto_heading()
         attitude_control->input_euler_angle_roll_pitch_yaw(wp_nav->get_roll(), wp_nav->get_pitch(), auto_yaw.yaw(), true);
     }
+
+    if (guided_mode == Guided_CircleMoveToEdge) {
```

Comment:
why check here ? just move it into the Guided_CircleMoveToEdge run() case instead.

Reply:
@khancyr  I have relocated the check to Guided_CircleMoveToEdge run() case.